README.md

```diff
@@ -4,79 +4,266 @@ Boost is a tool for Filecoin storage providers to manage data storage and retrie
+
+3. Build images
+
+```
+cd docker
+make build/all lotus_version=1.17.1-rc2 boost_version=1.3.0-rc1
 ```
 
-5. Interact with Boost
+### Start devnet docker stack
 
-Pass the client address (wallet) and the provider address to the `dummydeal` command.
-Note that
-- the client address is the address of a wallet with funds in `lotus wallet list`
-- you can find the provider address in `~/.boost/config.toml` under the config key `Wallets.Miner`
+1. Run
 
 ```
-boostd dummydeal <client address> <provider address>
+docker compose up -d
 ```
 
-## Running the UI in Development Mode:
+It will spin up `lotus`, `lotus-miner`, `boost`, `boost-gui` and `demo-http-server` containers. All temporary data will be saved in `./data` folder.
+
+The initial setup could take up to 20 min or more as it needs to download Filecoin proof parameters. During the initial setup, it is normal to see error messages in the log. Containers are waiting for the lotus to be ready. It may timeout several times. Restart is expected to be managed by `docker`.
+
+2. Try opening the Boost GUI http://localhost:8000 . Devnet is ready to operate when the URL opens and indicates no errors on the startup page.
+
+You can inspect the status using `docker compose logs -f`.
 
-1. Run the server
+### Making a deal
+
+The `boost` container is packed with `boost` and `lotus` clients. You can connect to the container with the command `docker compose exec boost /bin/bash` and follow instructions for [storing files with Boost guide](https://boost.filecoin.io/tutorials/how-to-store-files-with-boost-on-filecoin). But the recommended startup is to follow the semi-interactive demo first:
 
 ```
-cd react
-npm install
-npm start
+# Attach to a running boost container
+docker compose exec boost /bin/bash
+
+# Execute the demo script /app/sample/make-a-deal.sh
+root@83260455bbd2:/app# ./sample/make-a-deal.sh
 ```
 
-2. Open UI
+### Accessing Lotus from localhost
 
+By default the [docker-compose.yaml](./docker-compose.yaml) does not expose any port of the `lotus` container. To access the `lotus` from a local machine:
+1. You can either expose `1234` in [docker-compose.yaml](./docker-compose.yaml) or find the IP of the `lotus` container using `docker inspect lotus | grep IPAddress` command.
+2. Get the `FULLNODE_API_INFO`
 ```
-http://localhost:3000
+docker exec -it lotus lotus auth api-info --perm=admin
+FULLNODE_API_INFO=eyJ...ms4:/dns/lotus/tcp/1234/http
 ```
+3. Change the `dns/lotus/tcp/1234/http` to `ip4/<127.0.0.1 or container's IP>/tcp/1234/http` for the use in `FULLNODE_API_INFO`.
 
-## Running a devnet:
+### Cleaning up
 
-Follow the instructions in the [devnet guide](./documentation/devnet.md)
+To stop containers and drop everything:
+```
+docker compose down --rmi local
+
+rm -rf ./data
+```
 
```

Comment:
Not sure, but maybe we need also the instruction to delete `/var/tmp/filecoin-proof-parameters`?